src/viam/sdk/services/mlmodel/server.cpp

```diff
@@ -34,7 +38,7 @@ void MLModelServiceServer::register_server(std::shared_ptr<Server> server) {
 ::grpc::Status MLModelServiceServer::Infer(
     ::grpc::ServerContext* context,
     const ::viam::service::mlmodel::v1::InferRequest* request,
-    ::viam::service::mlmodel::v1::InferResponse* response) {
+    ::viam::service::mlmodel::v1::InferResponse* response) noexcept try {
```

Comment:
Exceptions should never escape up to grpc. Mark the function noexcept and use a function-scope try block to ensure that exceptions are always handled.